<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>315</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>200</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(80, 80, 80);

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button__digit__1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>115</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>115</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>115</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>165</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>165</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>165</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>215</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>215</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);
</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>215</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>115</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_multiply">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>215</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_minus">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>165</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_divide">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>265</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>65</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button__digit__0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>265</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_point">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>265</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>1</y>
      <width>240</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(100,100,100);
color:#fff;
font-size:20px;
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="indent">
     <number>4</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plusandminus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>65</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>50</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(97, 97, 97);
color:#fff
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <zorder>button_multiply</zorder>
   <zorder>button_plus</zorder>
   <zorder>button__digit__1</zorder>
   <zorder>button__digit__2</zorder>
   <zorder>button__digit__3</zorder>
   <zorder>button__digit__4</zorder>
   <zorder>button__digit__5</zorder>
   <zorder>button__digit__6</zorder>
   <zorder>button__digit__7</zorder>
   <zorder>button__digit__8</zorder>
   <zorder>button__digit__9</zorder>
   <zorder>button_minus</zorder>
   <zorder>button_plus</zorder>
   <zorder>button_minus</zorder>
   <zorder>button_multiply</zorder>
   <zorder>button_divide</zorder>
   <zorder>button_equal</zorder>
   <zorder>button__digit__0</zorder>
   <zorder>button_point</zorder>
   <zorder>button_clear</zorder>
   <zorder>label</zorder>
   <zorder>button_plusandminus</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
